pkg/kube/controllers/statefulset/statefulset_webhook.go

```diff
@@ -0,0 +1,128 @@
+package statefulset
```

Comment:
should live in the `quarksstatefulset` package

Reply:
see above